app/controllers/idv/in_person/ready_to_verify_controller.rb

```diff
@@ -12,8 +12,8 @@ class ReadyToVerifyController < ApplicationController
 
       check_or_render_not_found -> { IdentityConfig.store.in_person_proofing_enabled }
 
-      before_action :handle_fraud
       before_action :confirm_two_factor_authenticated
```

Comment:
The fix makes sense.  Could you please add a test for it?

Reply:
Test added